README.md

```diff
@@ -29,6 +28,8 @@ appcenter_upload(
 )
 ```
 
+The action parameters `api_token` and `owner_name` can also be omitted when their values are [set as environment variables](https://docs.fastlane.tools/advanced/#environment-variables) `APPCENTER_API_TOKEN` and `APPCENTER_OWNER_NAME`. A detailed list of all available environment variables can be found in our [documentation](https://docs.microsoft.com/appcenter/build/custom/scripts/#app-center-variables).
```

Comment:
@derpixeldan These are not the same environment variables as what is documented on https://docs.microsoft.com/appcenter/build/custom/scripts/#app-center-variables . This paragraph was about env variables used to configure the `fastlane` plugin, not Build 🙂

Reply:
@lumaxis ➕